Task7/vertex-ai-qwikstart/online-retail-clv-3M/cloudbuild.yaml

```diff
@@ -0,0 +1,5 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'us-central1-docker.pkg.dev/vertex-ai-project2/online-retail-clv/dnn-regressor:latest', '.' ]
+images: 
+- 'us-central1-docker.pkg.dev/vertex-ai-project2/online-retail-clv/dnn-regressor:latest'
```

Comment:
Are you using US region in everything?

Reply:
That was the default set by the qwiklab, now fixed to europe-west1 and made sure that the notebook still runs as expected